portfolio/src/main/webapp/script.js

```diff
@@ -36,7 +21,7 @@ function addRandomFact() {
       'I can still recite most of Hamlet\'s third soliloquy.', 
       'I have 7 pets.', 
       'I can dance a jig.'];
-
+    console.log(facts[2]);
```

Comment:
If everything is looking good, this log should be removed. Logging is great for debugging but, once thing are working, we don't want to push debugging to master.

Reply:
Sounds good, that's removed